src/test/java/it/gov/pagopa/payment/connector/rest/reward/AuthPaymentMapperTest.java

```diff
@@ -0,0 +1,94 @@
+package it.gov.pagopa.payment.connector.rest.reward;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import it.gov.pagopa.payment.connector.rest.reward.dto.AuthPaymentRequestDTO;
+import it.gov.pagopa.payment.connector.rest.reward.dto.AuthPaymentResponseDTO;
+import it.gov.pagopa.payment.connector.rest.reward.mapper.AuthPaymentMapper;
+import it.gov.pagopa.payment.dto.AuthPaymentDTO;
+import it.gov.pagopa.payment.enums.SyncTrxStatus;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+import it.gov.pagopa.payment.test.fakers.AuthPaymentResponseDTOFaker;
+import it.gov.pagopa.payment.test.fakers.TransactionInProgressFaker;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+ class AuthPaymentMapperTest {
+
+  private AuthPaymentMapper mapper;
+
+  @BeforeEach
+  void setUp() {
+    mapper = new AuthPaymentMapper();
+  }
+
+  @Test
+  void rewardMap() {
+
+    TransactionInProgress transaction = TransactionInProgressFaker.mockInstance(1,
+        SyncTrxStatus.IDENTIFIED);
+    AuthPaymentRequestDTO result = mapper.rewardMap(transaction);
+
+    assertAll(() -> {
+      assertNotNull(result);
+      assertEquals(transaction.getId(), result.getTransactionId());
+      assertEquals(transaction.getUserId(), result.getUserId());
+      assertEquals(transaction.getMerchantId(), result.getMerchantId());
+      assertEquals(transaction.getSenderCode(), result.getSenderCode());
+      assertEquals(transaction.getMerchantFiscalCode(), result.getMerchantFiscalCode());
+      assertEquals(transaction.getVat(), result.getVat());
+      assertEquals(transaction.getIdTrxIssuer(), result.getIdTrxIssuer());
+      assertEquals(transaction.getTrxDate(), result.getTrxDate());
+      assertEquals(transaction.getAmountCents(), result.getAmountCents());
+      assertEquals(transaction.getAmountCurrency(), result.getAmountCurrency());
+      assertEquals(transaction.getMcc(), result.getMcc());
+      assertEquals(transaction.getAcquirerCode(), result.getAcquirerCode());
+      assertEquals(transaction.getAcquirerId(), result.getAcquirerId());
+      assertEquals(transaction.getIdTrxAcquirer(), result.getIdTrxAcquirer());
+      assertEquals(transaction.getOperationTypeTranscoded(), result.getOperationType());
+      assertEquals(transaction.getTrxChargeDate(), result.getTrxChargeDate());
+      assertEquals(transaction.getCorrelationId(), result.getCorrelationId());
+      assertEquals(transaction.getHpan(), result.getHpan());
+    });
+  }
```

Comment:
use it.gov.pagopa.payment.test.utils.TestUtils#checkNotNullFields method to verify that there are not other fields null (this way, future changes to the models, but not mapped, would be intercepted by this test)